legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -75,7 +73,7 @@ def get_businesses(identifier: str):
     q_account = request.args.get('account')
     current_app.logger.info('account info request, for account: %s', q_account)
     if q_account and jwt.has_one_of_roles([SYSTEM_ROLE, 'account_identity']):
-        token = jwt.get_token_auth_header()
+        # token = jwt.get_token_auth_header()
```

Comment:
please uncomment L76

Reply:
The token in the line is not being used, giving pylint error, so I’m removed this line.